Lib/defcon/tools/unicodeTools.py

```diff
@@ -2,2954 +2,7 @@
 from __future__ import unicode_literals
 import unicodedata
```

Comment:
since you do `from fontTools import unicodedata` in the lines below, this line should be removed.